automation.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+from flask_init import app
+class Tester(unittest.TestCase):
+	def test_init_server(self):
```

Comment:
did you test this when the application was not running?

Reply:
Yes, using 'test_client' on 'app' (flask_init) will initialize the server and check if the return code is 200.